### PR TITLE
[SigV4] Documenation Changes in Sigv4

### PR DESCRIPTION
**Pull request by gshvang**
This PR contains changes to pages.dox file for documentation.

### CODE REVIEW

File: docs/doxygen/pages.dox [hunk 1]
```diff
@@ -3,6 +3,13 @@
 @anchor sigv4
 @brief AWS Iot SigV4 Utility
 
+<p>
+The AWS IoT SigV4 Library is a standalone utility for generating a signature and
+authorization header according to the specifications of the Signature Version 4
```
Comment (muneebahmed10) [inline docs/doxygen/pages.dox:8]: Nit
```suggestion
authorization header according to the specifications of the AWS Signature Version 4
```

File: docs/doxygen/pages.dox [hunk 2]
```diff
@@ -16,6 +23,15 @@
 All functions in the SigV4 library operate only on the buffers provided and use only
 local variables on the stack.
 </p>
+
+<h3>Compliance & Coverage</h3>
+
+<p>
+The SIGV4 library is designed to be compliant with ISO C90 and MISRA C:2012.
+All functions are written to have minimal complexity. Unit tests and CBMC proofs
```
Comment (muneebahmed10) [inline docs/doxygen/pages.dox:31]: Is this wording the same as other libraries? "Minimal complexity" seems kind of vague

Comment (gshvang) [reply docs/doxygen/pages.dox:31]: Yeah this is same as other spoke repos. I took reference from coreJSON Library.

**Review by muneebahmed10 — APPROVED** [review]

File: docs/doxygen/pages.dox [hunk 3]
```diff
@@ -3,6 +3,13 @@
 @anchor sigv4
 @brief AWS Iot SigV4 Utility
 
+<p>
+The AWS IoT SigV4 Library is a standalone utility for generating a signature and
+authorization header according to the specifications of the AWS Signature Version 4
+signing process. This utility is an optional addition to applications sending direct
+HTTP requests to AWS services requiring SigV4 authentication. The library is written in C and designed to be compliant with ISO C90 and MISRA C. It has proven safe memory use.
```
Comment (aggarg) [inline docs/doxygen/pages.dox:10]:
```suggestion
HTTP requests to AWS services requiring SigV4 authentication. The library is written
in C and designed to be compliant with ISO C90 and MISRA C. It has proven safe
memory use.
```

**Review by aggarg — APPROVED** [review]

**Pull request merged**